.github/workflows/stale.yml

```
@@ -0,0 +1,33 @@
+name: Stale Check
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 0 * * *'
```

Let's randomize this so it's not at midnight UTC.

```suggestion
    - cron: '16 22 * * *'
```